src/main/java/space/obminyashka/items_exchange/model/Location.java

```diff
@@ -12,6 +12,11 @@
 @Getter
 @Setter
 @EqualsAndHashCode(exclude = {"id", "advertisements"})
+@Table(name = "location",
```

Comment:
parameter "name" could be omitted because it's became mandatory only when the table and the entity have different names.